packages/editor-ui/src/components/OutputPanel.vue

```diff
@@ -209,6 +215,29 @@ const canPinData = computed(() => {
 	return pinnedData.isValidNodeType.value && !props.isReadOnly;
 });
 
+const allToolsWereUnusedNotice = computed(() => {
+	if (!node.value || runsCount.value === 0) return undefined;
+
+	// With pinned data there's no clear correct answer for whether
+	// we should use historic or current parents, so we don't show the notice,
+	// as it likely ends up unactionable noise to the user
+	if (pinnedData.hasData.value) return undefined;
+
+	const toolsAvailable = props.workflow.getParentNodes(
+		node.value.name,
+		NodeConnectionType.AiTool,
+		1,
+	);
+	const toolsUsedInLatestRun = toolsAvailable.filter(
+		(x) => !!workflowRunData.value?.[x]?.[props.runIndex],
```

Comment:
Please use descriptive names where possible, because it's clearer to the reader.
```suggestion
		(tool) => !!workflowRunData.value?.[tool]?.[props.runIndex],
```